### PR TITLE
using setuptools.find_packages to wrap modules

### PR DESCRIPTION
**Pull request by brunobord**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1,2 @@
 include README.rst
+prune workalendar/tests*
```
Comment (Natim) [inline MANIFEST.in]: Strange way do to that. I though adding tests in the package was to ship them as part of the code.
If you do not want to ship them you should probably put them outside the package.

**Comment by Natim** [comment]
I like the fact that you use find_package I don't like the fact that you exclude tests from the package that way.

File: MANIFEST.in (hunk 1 above)
Comment (brunobord) [inline MANIFEST.in]: I wasn't sure I could still run the tests effectively, but I may try to remove the test directory from the package, right.

respectfully, I can't see any use-case where you would need to import test packages at runtime. If you "pip install" stuff, you probably just want the part that works. Tests are just meant to be imported / loaded in a test env, but they're probably not a *runtime* package.

```python
>>> from workalendar.tests.test_europe import FranceTest
>>> # Then what?
```

Comment (Natim) [inline MANIFEST.in]: > I can't see any use-case where you would need to import test packages at runtime.

When your python package is packaged for Debian, they want to run the tests to see if the package is correctly packages.

    py.test workalendar

**Comment by brunobord** [comment]
a bit confused by this: https://github.com/novafloss/workalendar/blob/find-packages/setup.py#L10-L13

shall I drop it?

**Comment by Natim** [comment]
Yes you can drop the distutils one.

**Comment by Natim** [comment]
It looks good to me :+1:

**Pull request merged**